packages/scripts/utils/config.js

```diff
@@ -17,7 +17,24 @@ const hasJestConfig = () =>
 	hasProjectFile( 'jest.config.json' ) ||
 	hasPackageProp( 'jest' );
 
+const hasWebpackConfig = () => hasCliArg( '--config' ) ||
+	hasProjectFile( 'webpack.config.js' ) ||
+	hasProjectFile( 'webpack.config.babel.js' );
+
+const getWebpackArgs = ( additionalArgs = [] ) => {
+	const webpackArgs = [];
+	if ( hasWebpackConfig() ) {
+		webpackArgs.push( ...getCliArgs() );
+	} else {
+		webpackArgs.push( ...[ '--config', fromConfigRoot( 'webpack.config.js' ) ] );
```

Comment:
Does it skip all CLI arguments provided on purpose? Noting, that in other supported scripts we never silence additional params passed to the script.

Reply:
6fc1778